sample/F8/tracks

```diff
@@ -0,0 +1,202 @@
+[
+   {
+      "id":1,
+      "name":"ADS & MONETIZATION",
+      "color":"#BF532E",
+      "description":null,
+      "location":"C",
```

Comment:
description and location can be removed from track object. Shouldn't exist and not used anywhere either.